build.fsx

```diff
@@ -611,17 +617,17 @@ open Fake.Core.TargetOperators
 "BuildTests"
   ==> "DotnetTests"
 
-"BuildTests"
-  ==> "RunTests"
+//"BuildTests"
+  //==> "RunTests"
   ==> "All"
 
-"GenerateDocs" ==> "All"
+//"GenerateDocs" ==> "All"
```

Comment:
Does GenerateDocs still work?

Reply:
I check..

Reply:
I have to make some changes. Figuring out how this works

Reply:
Ok this is working. I push 👍